sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryColumn,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { Message } from "./message.model";
+import { RecordDataModel } from "./record.model";
+import { User } from "./user.model";
+
+@Entity({
+  name: TableNames.Notifications,
+})
+export class Notification extends RecordDataModel {
+  /**
+   *  Primary key column.
+   */
+  @PrimaryColumn({
+    name: "id",
+    nullable: false,
+  })
+  id: number;
+  /**
+   * User associated with this notification.
+   */
+  @OneToOne(() => User, { eager: false, cascade: false })
+  @JoinColumn({
+    name: "user_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  user: User;
+  /**
+   * Template id.
+   */
+  @Column({
+    name: "template_id",
+    nullable: false,
+  })
+  templateId: string;
+  /**
+   * GC notification payload.
+   */
+  @Column({
+    name: "gc_notify_payload",
+    type: "jsonb",
+    nullable: false,
+  })
+  messagePayload: unknown;
```

Comment:
I do not see a column called gc_notify_payload in the create table script.
It is called message_payload

![image](https://user-images.githubusercontent.com/54600590/195664562-c3c3bd47-f89e-49b4-b9c5-c340e692eb91.png)